### PR TITLE
Support crypto 2.0.0.

### PR DESCRIPTION
**Pull request by nex3**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # Changelog
 
 ## 0.3.0+4
```
Comment (jakemac53) [inline CHANGELOG.md]: 0.3.0+5

**Comment by jakemac53** [comment]
Lgtm, just the one nit

**Comment by jakemac53** [comment]
Just merged in another change for 0.3.0+5, please just add this message to the 0.3.0+5 changelog after rebasing.

**Comment by nex3** [comment]
@jakemac53 Done, PTAL.

**Pull request merged**